packages/twenty-server/src/engine/metadata-modules/flat-field-metadata/utils/compare-two-flat-field-metadata.util.ts

```diff
@@ -73,15 +82,27 @@ export const compareTwoFlatFieldMetadata = ({
       }
 
       // Remove below assertion when we authorize relation edition, see https://github.com/twentyhq/twenty/commit/39f6f3c4bb101272a9014e142a842d0801a3c33b
-      if (
+      const isRelationFieldType =
```

Comment:
a bit lost by naming here: compareFieldMetadataOptions ? it does not seem related to options

Reply:
Sorry but I'm not sure to understand

Reply:
It's the transfarm metadata to compare options, this method ignore a given property dynamically